### PR TITLE
Created 2022_Happy_Eight_Day.md

### PR DESCRIPTION
**Pull request by Harsh-Tewari**
This is my first contribution to open source ...

### CODE REVIEW

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for submitting this pull request! There are some changes that need to be made in order to format this correctly. Please consult our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) if you need extra guidance. Once you amend this pull request, we can merge it in.

Please also commit to the main branch.

*Please note: after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by ltagliaferri** [comment]
Please open a new pull request that references the main branch.

**Comment by Harsh-Tewari** [comment]
Thanks

**Pull request closed without merging**
